code/modules/shuttles/shuttle.dm

```diff
@@ -215,6 +215,11 @@
 						continue
 					TA.ChangeTurf(ceiling_type, TRUE, TRUE, TRUE)
 
+	for(var/obj/structure/shuttle_part/part in get_area(current_location))
```

Comment:
Shuttles can be multiz so you'll need to check areas in shuttle_areas then do that